src/classes/CW_CampaignRewardTriggerHandlerTest.cls

```diff
@@ -0,0 +1,68 @@
+/**
+* File Name   :    CW_WizardRewardUtil .cls
+* Description :    Select rewards and triggers
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.3 28/07/2014 Cristina P.	B-122940 Campaign Reward - Many Rewards per Trigger
+*
+*/
+@isTest
+private class CW_CampaignRewardTriggerHandlerTest {
+
+    static testMethod void myUnitTest() 
```

Comment:
Style issue: test methods should be annotated with @IsTest for better readibility